playbooks/roles/edxapp/templates/lms_gunicorn.py.j2

```diff
@@ -64,26 +64,36 @@ def _should_sample(trace_id, sample_rate):
 def sampler(fields):
     """
     the goal is to default to sending everything (ie, sample rate = 1)
-
     *except*, the requests for `/heartbeat`, GCP load balancer hits about
     4 times per second, we want to aggressively downsample
 
     and `/event`, which just sends an event to segment.io in the background
     which *can* be interesting, but doesn't directly impact performance
     or user experience and effectively doubles our honeycomb event rate,
     so we also sample that down
+
+    since we've been hitting our quota, we also need to downsample
+    "boring" requests, which is basically anything that was successful
+     and fast. Those requests are unlikely to be interesting to us
+    for investigating bugs or performance problems.
     """
 
     sample_rate = 1
     heartbeat_sample_rate = 100
     event_sample_rate = 10
+    boring_sample_rate = 100
 
     path = fields.get('request.path', fields.get('app.request.path'))
+    response_code = int(fields.get('response.status_code', 404))
+    duration = float(fields.get('duration_ms', 1000))
 
     if path == "/heartbeat":
         sample_rate = heartbeat_sample_rate
     if path == "/event":
         sample_rate = event_sample_rate
+    # a 200 OK response in less than 100ms is "boring"
+    if response_code == 200 and duration < 100.0:
```

Comment:
I checked honeycomb I was surprised to learn that we have requests that finishes under 100ms.

Reply:
Yeah, mostly `/events` (which we already sample, but will now be sampling those even more aggressively) and various static assets (I'm assuming that the ones that are served in < 100ms are small and basically being cached in memory somewhere).